src/content/getting-started/architecture/route-agent/_index.en.md

```diff
@@ -33,6 +33,41 @@ the new active Gateway Engine node.
 
 ## OVN Kubernetes
 
-For the OVN Kubernetes CNI plugin, host network routing is configured on all nodes and,
-on the active Gateway node, IP forwarding is configured between the `ovn-k8s-gw0`
-and cable interfaces.
+ With OVN Kubernetes we reuse the GENEVE tunnels created by OVNKubernetes CNI to reach the
+gateway nodes from non-gateway nodes and a separate VXLAN tunnel is not created.
+
+With OVN we can have two deployment models
+
+### Single Zone
+
+A single-zone deployment involves a single OVN database and a set of master nodes that
+program it.
+
+Here, Submariner configures the ovn_cluster_router to route traffic to other clusters through the
```

Comment:
In the previous diagram (i.e., ovn-submariner-architecture.svg), we captured the sample OVN LRPs and LRs and it served as a good reference diagram to understand the architecture. Can you please add a similar one for the OVN-IC?

Reply:
Added.

Reply:
Sorry missed it , updated now.